base/applications/regedit/lang/ro-RO.rc

```diff
@@ -31,17 +31,17 @@ Opțiuni:\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
-  /C             Importă conținutul unui fișier de regiștri.\n\
-  /D             Șterge o cheie de regiștrii specificată.\n\
-  /E             Exportă conținutul unei cheie de regiștri specificată într-un fișier.\n\
+  /C             Importă conținutul unui fișier de regiștri. Este importat întregul registru dintr-un fișier.\n\
+  /D             Șterge o cheie de registru specificată.\n\
+  /E             Exportă conținutul unei chei de registru specificată într-un fișier. Este exportat întregul registru într-un fișier.\n\
```

Comment:
The same kind of comment as above - the addition (starting with "Este") doesn't make sense. Other than that, you changed it for the better.